MergePythonSDK/accounting/.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+# ref: https://docs.gitlab.com/ee/ci/README.html
```

Comment:
I'm not sure where this came from, perhaps the generator default files. If so please add them to the `.openapi-generator-ignore` file so it stops getting generated since we don't use gitlab.